src/views/Pipeline.vue

```diff
@@ -546,10 +551,8 @@ export default defineComponent({
     },
     async updatePinnedJobs(enumId: any) {
       const pinnedJobs = new Set(this.getPinnedJobs);
-      if(pinnedJobs.has(enumId)) {
+      if (pinnedJobs.has(enumId)) {
         pinnedJobs.delete(enumId);
-      } else {
```

Comment:
Why this is removed?

Reply:
this was removed because of the pipeline.vue page has not support to pin job this Method(updatePinnedJobs) is only used for unpin jobs